torch/csrc/distributed/c10d/ProcessGroupNCCL.hpp

```diff
@@ -1221,16 +1243,7 @@ class TORCH_API ProcessGroupNCCL : public Backend {
   std::set<c10::DeviceIndex> usedDeviceIdxs_;
 
   // Flag to denote if a coalescing groupStart/groupEnd block is active
-  int coalescing_state_ = 0;
-
-  // Stores device indexes for all collectives run inside a coalescing block
-  at::Device coalescedDevice_ = at::Device("cuda");
-
-  // Stores communicators for all collectives run inside a coalescing block
-  std::shared_ptr<NCCLComm> coalescedComm_ = nullptr;
-
-  // Whether the coalesced calls are sync or async.
-  bool coalescedAsync_;
+  struct CoalescingState coalescingState_;
```

Comment:
Do you think if we call it `coalescingMode_` would be better? No strong opinion.